java-frontend/src/main/java/org/sonar/java/cfg/CFG.java

```diff
@@ -81,7 +81,13 @@
 
 public class CFG {
 
-  private final Symbol.MethodSymbol methodSymbol;
+  private final boolean ignoreBreakAndContinue;
+
+  public void setMethodSymbol(Symbol.MethodSymbol methodSymbol) {
```

Comment:
Even if [S1213](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1213) is not enabled, I would prefer to have this setter a bit lower in the members declarations (at least after constructors).